### PR TITLE
Segment the ISICs data set with the U-net - 45051423

### PR DESCRIPTION
**Pull request by Thomas-TTC**
Task 3 : Segmentation of ISIC dataset using U-net

Student name: Thomas Chen
Student number: 45051423

### CODE REVIEW

**Comment by SiyuLiu0329** [comment]
- Some commit messages should be more specific
- WRONG source and target branches. DO NOT use master
- Code valid for Easy Difficulty
- Results appear ok

Other Feedback
- Convert links to citation in the future
- main file is completely uncommented

**Comment by rexsw** [comment]
Code looks good for easy difficulty

For next time;

Code in files like driver scripts should be in a main function

Tensorflow has functions for data generators and more so there’s no need to also use pandas in your project. Same for sklearn and train/test splitting plus cross vaildation.

File: recognition/ISICs_Unet/data.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
+import numpy as np
```
Comment (shakes76) [inline recognition/ISICs_Unet/data.py:1]: Code header block should be added for all scripts/source files

**Comment by shakes76** [comment]
## TF Usage
OK
## Good Practice (Design/Commenting)
Good design and comments
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage, Readme OK
Module present
## Commit Log
Meaningful commit messages 
Progressive commits used 
## Documentation
Description and Comments good
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Incorrect Branch) 
**Feedback required. Please update the PR to the correct branch to be used topic-recognition.**
No Request Description. In the future, a more detailed pull request could've been done. When developer receiving the request is not known, the request could be rejected. Remember you could use/reuse the README info.

File: recognition/ISICs_Unet/data.py (hunk 1 above)
Comment (Thomas-TTC) [reply recognition/ISICs_Unet/data.py:1]: Code header blocks are added in the new pull request to correct branch

**Comment by Thomas-TTC** [comment]
Thanks for feedbacks,

Problems are solved in my new pull request, including

1. pull request to correct branch and add more detail to it
2. comment on the main files
3. code header blocks are added
4. convert links to citations in README

**Comment by shakes76** [comment]
Files look good but the changes are still going into master branch, please update to topic-recognition.

**Comment by Thomas-TTC** [comment]
I actually make another pull request yesterday, with my branch also updated to topic-recognition.

I'm not sure if merging from my master branch is ok.

**Comment by shakes76** [comment]
Updated PR submitted, closing this one.

**Pull request closed without merging**